tests/test_test_driver.py

```
@@ -521,5 +521,17 @@ def test_stress_and_pressure_variables():
     assert td._get_cell_cauchy_stress() == [-1, -1, -1, 0, 0, 0]
 
 
+def test_kim_model_name():
+    modelname = "LJ_ElliottAkerson_2015_Universal__MO_959249795837_003"
+    td = TestInitKIMTestDriver(modelname)
+    assert td.kim_model_name == modelname
+    td = TestInitKIMTestDriver(LennardJones())
+    try:
+        td.kim_model_name
```

Ruff (0.15.12)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)